test/functional/feature_assumevalid.py

```diff
@@ -134,8 +132,7 @@ def run_test(self):
 
         # Bury the assumed valid block 2100 deep
         for _ in range(2100):
-            block = create_block(self.tip, create_coinbase(height), self.block_time)
-            block.nVersion = 4
+            block = create_block(self.tip, create_coinbase(height), self.block_time, version=4)
```

Comment:
any reason to set `4`? This is already the default and shouldn't matter either way?